gooddata-sdk/gooddata_sdk/catalog/workspace/service.py

```diff
@@ -2,16 +2,23 @@
 from __future__ import annotations
 
 import functools
+from pathlib import Path
 from typing import List, Union
 
+import yaml
+
 import gooddata_afm_client.apis as afm_apis
 import gooddata_afm_client.models as afm_models
 import gooddata_metadata_client.apis as metadata_apis
 from gooddata_metadata_client.exceptions import NotFoundException
 from gooddata_sdk.catalog.types import ValidObjects
-from gooddata_sdk.catalog.workspace.model.content_objects.dataset import CatalogFact, CatalogLabel
-from gooddata_sdk.catalog.workspace.model.content_objects.metric import CatalogMetric
-from gooddata_sdk.catalog.workspace.model.workspace import CatalogWorkspace
+from gooddata_sdk.catalog.workspace.declarative_model.workspace.workspace import (
```

Comment:
RIght now you can expose also analytics and ldm models too.
From business point of view it make sense to deliver them separately.